src/test/javascript/spec/integration/code-editor/code-editor-container.integration.spec.ts

```diff
@@ -282,10 +268,10 @@ describe('CodeEditorContainerIntegration', () => {
         const commitState = CommitState.UNDEFINED;
```

Comment:
This looks like a code duplication from the cleanInitialize, or did I miss some differences?

Reply:
The commitState in the cleanInitialize cannot be used in this method therefore it is redeclared here. I did some refactoring here nevertheless, since this const is only used once.